pair.go

```diff
@@ -93,9 +96,13 @@ func (cli *Client) handlePairSuccess(node *waBinary.Node) {
 	jid, _ := pairSuccess.GetChildByTag("device").Attrs["jid"].(types.JID)
 	lid, _ := pairSuccess.GetChildByTag("device").Attrs["lid"].(types.JID)
 	platform, _ := pairSuccess.GetChildByTag("platform").Attrs["name"].(string)
+	clientPropsBytes := pairSuccess.GetChildByTag("client-props").Content.([]byte)
+
+	var props waCompanionReg.ClientPairingProps
+	err := proto.Unmarshal(clientPropsBytes, &props)
```

Comment:
This error seems to be ignored. `_` should be used instead if the error doesn't matter, but alternatively it could be logged